app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  belongs_to :class
```

Comment:
Same here with multiple parent dependencies as mentioned in the previous comment about polymorphism. https://guides.rubyonrails.org/association_basics.html#polymorphic-associations. Would make it easier to manipulate later on if we have to.